unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/CharacterController/DCLCharacterController.cs

```diff
@@ -268,7 +264,7 @@ internal void LateUpdate()
         {
             velocity.x = 0f;
             velocity.z = 0f;
-            velocity.y += gravity * deltaTime;
+            velocity.y += gravity * Time.deltaTime;
```

Comment:
IIRC, the reason of capping the `deltaTime` was to avoid the player "teleporting" in cases of big hiccups. So maybe it would be wise to keep deltaTime capping logic but with a bigger threshold?

Reply:
We can talk about this in the daily/post daily today

Reply:
What would cause the teleporting? 

For the local player, In a hiccup the engine will stop processing input, wont it?
For another player the position won't be sent to kernel and in any case we have the interpolation.

Am I missing something?

We could also manually introducing a big hiccup and check the behaviour.

Reply:
We already talked about it and concluded that this may not be a big enough issue to justify the cap for now. We are going to test this further to be sure. Resolving this conversation.